docs/code.rst

```diff
@@ -132,6 +132,9 @@ Variable                            Description
 ``{{ latest_date }}``               same as ``{{ ds }}``
 ``{{ ti }}``                        same as ``{{ task_instance }}``
 ``{{ params }}``                    a reference to the user-defined params dictionary
+``{{ var.value }}``                 global defined variables represented as a dictionary
+``{{ var.json }}``                  global defined variables represented as a dictionary
```

Comment:
Ah, I see based on the tests that its:

    {{ var.json.foo }}

The tests use a dictionary for the variable:

    {{ var.json.a_variable.foo }}

I assume it'll work with just a key/value pair as well?